src/Toml/Type/AnyValue.hs

```diff
@@ -31,7 +32,10 @@ instance Show AnyValue where
 instance Eq AnyValue where
     (AnyValue val1) == (AnyValue val2) = case sameValue val1 val2 of
         Right Refl -> val1 == val2
-        Left _ -> False
+        Left _     -> False
+
+instance NFData AnyValue where
+    rnf (AnyValue val) = rwhnf val
```

Comment:
I wonder, whether's it's possible to use `rnf` here instead of `rwhnf`? 🤔 Because `rwhnf` is not accurate enough... :(

Reply:
If `rnf` then 
-- deriving instance NFData (Value t)
-- deriving instance Generic (Value t)
and that entails
```
• Can't make a derived instance of ‘Generic (Value t)’:
    Bool must be a vanilla data constructor, and
    Integer must be a vanilla data constructor, and
    Double must be a vanilla data constructor, and
    Text must be a vanilla data constructor, and
    Date must be a vanilla data constructor, and
    Array must be a vanilla data constructor
```

Reply:
need some :ice_cream:

Reply:
@willbasky Yes, unfortunately, you can't derive `Generic` and `NFData` instances for GADTs. It's a known problem. But you can implement `NFData` instance manually. Like:

```
rnf (Integer n) = rnf n
rnf (Text t) = rnf t
...
```